doc/classes/InputEvent.xml

```diff
@@ -64,27 +65,28 @@
 			<argument index="0" name="action" type="String">
 			</argument>
 			<description>
-				Returns true if the given action is released (i.e. not pressed). Not relevant for the event types MOUSE_MOTION, SCREEN_DRAG and NONE.
+				Returns [code]true[/code] if the given action is released (i.e. not pressed). Not relevant for the event types MOUSE_MOTION, SCREEN_DRAG and NONE.
 			</description>
 		</method>
 		<method name="is_action_type" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns [code]true[/code] if current input event has a type, a concret implementation and not only the main [code]InputEvent[/code] class.
```

Comment:
In other descriptions, you've said "if this event has" but in this one you say "current".

What does "concrete implementation" even mean?

Returns [code]true[/code] if this input event's type is one of the [code]InputEvent[/code] constants.